src/configs/proposalTitles.json

```diff
@@ -1,54 +1,41 @@
 {

```

Comment:
Could you remove these changes, we should keep this update to specifically cache update PRs

Reply:
I was getting merge conflicts but I will try that